bin/pkg.ml

```diff
@@ -10,16 +10,19 @@ module Opam_repository = struct
   type t = { url : OpamUrl.t }
 
   let default =
-    { url = OpamUrl.of_string "git+https://github.com/ocaml/opam-repository.git"
-    }
+    { url = OpamUrl.of_string "https://opam.ocaml.org/index.tar.gz" }
+
+  let is_tarball name =
+    String.is_suffix ~suffix:".gz" name || String.is_suffix ~suffix:".bz" name
```

Comment:
Nitpick but you can totally have non-tarball .gz files. Should this check for ".tag.gz" and ".tgz" instead of just ".gz"?

Reply:
Yeah, and also `.tar.bz2`, `.tbz` and `.zip`, they should all be supported by OPAM. I've updated the code!